test 6. Worker Overloading/expectedOut.txt

```diff
@@ -0,0 +1,113 @@
+Activities
+(9003, 20000, 101, 20200101)
+(9006, 20000, 102, 20200101)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9006, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+(9003, -1000, 1003, 20201202)
+Coffee stands
+(11, 'Bld-90', 1)
+(12, 'Bld-32', 2)
+(13, 'Bld-71', 1)
+Employees
+(1001, 'Alice', 45.5, 11)
+(1002, 'Bob', 45.5, 12)
+(1003, 'Charlie', 45.5, 12)
+(1004, 'Dan', 45.5, 13)
+Products
+(9001, 'Latte', 6.5, 0)
+(9002, 'Water', 7.0, 0)
+(9003, 'Magnum', 8.5, 0)
+(9004, 'Salad', 21.0, 0)
+(9005, 'Cafe & Mahafe', 11.5, 0)
+(9006, 'Chocolate', 8.5, 0)
+(100293, 'Ice Tea', 100.0, 0)
+Suppliers
+(101, 'Strauss', '*6860 49 Hasivim St. Petach Tikva')
+(102, 'Osem', '1-700-70-76-76')
+
+Employees report
+Alice 45.5 Bld-90 0
+Bob 45.5 Bld-32 0
+Charlie 45.5 Bld-32 340000.0
```

Comment:
how did you calculate it? manually or with python code?

Reply:
In my head, It's 8.5*40,000 which is just 34*10,000 or 340,000

Reply:
ok cool, just wanted to make sure (: